### PR TITLE
chore: replace the 'Get started' CTA with 'New source'

### PR DESCRIPTION
**Pull request by amlannandy**
### Summary

Replaced the "Get started" CTA with "New source" in the onboarding flow.

#### Related Issues / PR's

#6535 

#### Screenshots

Screenshot 2024-12-05 at 12 44 31 PM

#### Affected Areas and Manually Tested Areas

Only modified at a single place (button) in the onboarding flow.

----

> [!IMPORTANT]
> Replaces 'Get started' with 'New source' in `SideNav.tsx`, updating button text, icon, localization, styles, and tests.
> 
>   - **Behavior**:
>     - Replaces "Get started" button text with "New source" in `SideNav.tsx`.
>     - Updates button icon to `PackagePlus` in `SideNav.tsx`.
>   - **Localization**:
>     - Adds `"new_source": "New source"` to `onboarding.json`.
>   - **Styles**:
>     - Adjusts button alignment in `.continue-to-next-step` in `Onboarding.styles.scss`.
>   - **Tests**:
>     - Updates `InviteUserFlow.test.tsx` to check for "new_source" button text.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=SigNoz%2Fsignoz&utm_source=github&utm_medium=referral)<sup> for 8c6225185daadf0f5f24d0dc6319619264760ee7. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

File: frontend/src/container/OnboardingContainer/Onboarding.styles.scss [hunk 1]
```diff
@@ -99,6 +99,11 @@
 	display: flex;
 	justify-content: center;
 	margin: 48px 0;
+
+	> button {
```
Comment (amlannandy) [inline frontend/src/container/OnboardingContainer/Onboarding.styles.scss:103]: Lucid icons were not getting aligned properly with the text inside buttons. This styling is to manage that. Similar to how we have handled it in the "Invite" button in Line 190.

Comment (YounixM) [reply frontend/src/container/OnboardingContainer/Onboarding.styles.scss:103]: @amlannandy : You can use `periscope-btn` class to handle the alignment and styling 

Ex:

```
<Button
   className="periscope-btn primary"
   ...
>
    New Source
</Button>

```

Comment (amlannandy) [reply frontend/src/container/OnboardingContainer/Onboarding.styles.scss]: Ah got it. Have updated it now. Updated screenshot - 

Screenshot 2024-12-04 at 8 20 39 AM

**Comment by YounixM** [comment]
@amlannandy : The btn style is primary in the above screenshot, it should be default as per the design.

**Comment by amlannandy** [comment]
> @amlannandy : The btn style is primary in the above screenshot, it should be default as per the design.

Ah my bad. Have updated it now.

File: frontend/public/locales/en/onboarding.json [hunk 2]
```diff
@@ -4,5 +4,6 @@
 	"skip": "Skip",
 	"invite_user_helper_text": "Not the right person to get started? No worries! Invite someone who can.",
 	"select_use_case": "Select a use-case to get started",
-	"get_started": "Get Started"
+	"get_started": "Get Started",
```
Comment (YounixM) [inline frontend/public/locales/en/onboarding.json:7]: We can remove this if we don't use it anywhere

**Comment by amlannandy** [comment]
@YounixM Have the PR according to discussions with UX

**Review by YounixM — APPROVED** [review]

**Pull request merged**